### PR TITLE
Fix flaky test_to_from_pandas

### PR DESCRIPTION
**Pull request by eivindjahren**
Fix flaky test https://github.com/equinor/resdata/actions/runs/10628849221/job/29464486920

### CODE REVIEW

File: python/tests/rd_tests/test_rd_sum.py [hunk 1]
```diff
@@ -177,9 +177,9 @@ def test_to_from_pandas(summary):
         "TEST", baseline, dims=(smspec.nx, smspec.ny, smspec.nz)
     ).pandas_frame()
 
-    # round to hours
-    roundtrip.index = roundtrip.index.round(freq="H")
-    baseline.index = baseline.index.round(freq="H")
+    # round to days
```
Comment (berland) [inline python/tests/rd_tests/test_rd_sum.py:180]: Could we leave an explanation here for why we round (so much). Related to some int32 handling of dates?

Comment (eivindjahren) [reply python/tests/rd_tests/test_rd_sum.py:180]: Sure, its because there is so much conversion.

Comment (eivindjahren) [reply python/tests/rd_tests/test_rd_sum.py:180]: Updated

Comment (berland) [reply python/tests/rd_tests/test_rd_sum.py:180]: not pushed yet (?)

Comment (eivindjahren) [reply python/tests/rd_tests/test_rd_sum.py]: Forgot to push, is updated now.

**Review by berland — APPROVED** [review]
Great!

**Pull request merged**